slapp/transfers/upload.py

```diff
@@ -94,7 +93,8 @@ def run(self, db_conn: query_utils.DbConnection):
                     self.args['s3_bucket_name'],
                     prefix,
                     skip_keys=['full-video-source-ref']))
-            s3_manifests[-1].update(experiment_manifest)
+            s3_manifests[-1]['full-video-source-ref'] = \
```

Comment:
Calling this out as something that will probably have to change if/when the upload is parallelized. We should update the individual manifest object before adding it to the overall manifest list.

Reply:
:heavy_check_mark: